autoscheduler/frontend/src/tests/ui/Availability.test.tsx

```diff
@@ -122,7 +122,7 @@ describe('Availability UI', () => {
         .toHaveAttribute('aria-valuetext', expectedEnd);
     });
 
-    test('with an end time of 10 PM and a size of 30 mins if dragged below the bottom', async () => {
+    test('with an end time of LAST_HOUR-1 PM and a size of 30 mins if dragged below the bottom', async () => {
```

Comment:
```suggestion
    test('with an end time of LAST_HOUR PM and a size of 30 mins if dragged below the bottom', async () => {
```

Reply:
Since the expected end time is 11 PM and the LAST_HOUR is also 11 PM

Reply:
Great catch!